### PR TITLE
Add webpack sample

### PR DESCRIPTION
**Pull request by KonstantinTyukalov**
**Description:** This PR adds sample with `webpack 5`

### CODE REVIEW

File: samples/webpack/babel.config.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "presets": [
+        "@babel/preset-env"
+    ]
+}
```
Comment (vmapetr) [inline samples/webpack/babel.config.json:5]: nit: please add newline here

File: samples/webpack/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                    options: {
+                        presets: ["@babel/preset-env"],
+                    },
+                },
+            },
+        ],
+    },
+};
+
+module.exports = config
```
Comment (vmapetr) [inline samples/webpack/webpack.config.js:74]: And here too

**Review by vmapetr — APPROVED** [review]

File: samples/webpack/babel.config.json (hunk 1 above)
Comment (KonstantinTyukalov) [reply samples/webpack/babel.config.json]: Fixed, thanks!

File: samples/webpack/webpack.config.js (hunk 2 above)
Comment (KonstantinTyukalov) [reply samples/webpack/webpack.config.js]: Fixed, thanks!

**Review by alexander-smolyakov — APPROVED** [review]

**Review by wojgasior — APPROVED** [review]

**Comment by AndreyIvanov42** [comment]
Manually tested in Chrome, Edge, Firefox, Safari.

**Review by AndreyIvanov42 — APPROVED** [review]

**Pull request merged**